src/wwwroot/app/directives/datepickerwrapper/DatePickerTemplate.html

```diff
@@ -0,0 +1,3 @@
+   <datepicker date-format="dd-MM-yyyy" date-min-limit="01-01-1901" date-typer="true">
```

Comment:
try to use scope variable instead of const `dd-MM-yyyy` and date-min-limit (it must be configurable var)